### PR TITLE
fix: dont hardcode file manager and make use of $TERM

### PR DESCRIPTION
**Pull request by rubiin**
With this, user can use any file manager they have setup.
This also removes terminal and uses the one setup on $TERM varibale. The original PR contains a mistake as TERM env is used instead of TERMINAL

https://www.ibm.com/docs/en/informix-servers/14.10?topic=products-term-environment-variable-unix
https://bash.cyberciti.biz/guide/$TERM_variable

closes https://github.com/Jas-SinghFSU/HyprPanel/pull/156

### CODE REVIEW

**Comment by rubiin** [comment]
@Jas-SinghFSU check this . I think this can be merged

File: options.ts [hunk 1]
```diff
@@ -1160,8 +1160,6 @@ const options = mkOptions(OPTIONS, {
 
     scalingPriority: opt<ScalingPriority>('gdk'),
 
-    terminal: opt('kitty'),
-
     tear: opt(false),
```
Comment (Jas-SinghFSU) [inline options.ts:1163]:
```suggestion
    terminal: opt('$TERMINAL'),
    
    tear: opt(false),
```

File: modules/menus/dashboard/stats/index.ts [hunk 2]
```diff
@@ -7,7 +7,6 @@ import Cpu from 'services/Cpu';
 import Storage from 'services/Storage';
 import { renderResourceLabel } from 'customModules/utils';
 
-const { terminal } = options;
 const { enable_gpu, interval } = options.menus.dashboard.stats;
```
Comment (Jas-SinghFSU) [inline modules/menus/dashboard/stats/index.ts:10]:
```suggestion
const { terminal } = options;
const { enable_gpu, interval } = options.menus.dashboard.stats;
```

File: modules/menus/dashboard/stats/index.ts [hunk 3]
```diff
@@ -23,6 +22,11 @@ interval.connect('changed', () => {
     storageService.updateTimer(interval.value);
 });
 
+const handleClick = (): void => {
+    App.closeWindow('dashboardmenu');
+    Utils.execAsync(`bash -c "$TERMINAL -e btop"`).catch((err) => `Failed to open btop: ${err}`);
```
Comment (Jas-SinghFSU) [inline modules/menus/dashboard/stats/index.ts:27]:
```suggestion
    Utils.execAsync(`bash -c "${terminal} -e btop"`).catch((err) => `Failed to open btop: ${err}`);
```

Comment (Jas-SinghFSU) [reply modules/menus/dashboard/stats/index.ts:27]: We can leave these options in but have the default terminal configured in the options to '$TERMINAL'. That way if the user doesn't have a default terminal set, and the click doing nothing, they can instead configure it here to change that.

File: widget/settings/pages/config/general/index.ts [hunk 4]
```diff
@@ -34,12 +34,6 @@ export const BarGeneral = (): Scrollable<Child, Attribute> => {
                         themeOnly: false,
                     },
                 }),
-                Option({
-                    opt: options.terminal,
-                    title: 'Terminal',
-                    subtitle: "Tools such as 'btop' will open in this terminal",
-                    type: 'string',
-                }),
                 Option({
```
Comment (Jas-SinghFSU) [inline widget/settings/pages/config/general/index.ts:37]:
```suggestion
                Option({
                    opt: options.terminal,
                    title: 'Terminal',
                    subtitle: "Tools such as 'btop' will open in this terminal",
                    type: 'string',
                }),
                Option({
```

**Comment by rubiin** [comment]
@Jas-SinghFSU  fixed

**Review by Jas-SinghFSU — APPROVED** [review]

**Pull request merged**